.github/workflows/quest.yml

```
@@ -14,10 +14,10 @@ jobs:
   import:
     if: |
       github.event_name == 'workflow_dispatch' ||
-      github.event.label.name == 'reQUEST' ||
-      github.event.label.name == 'seQUESTered' ||
-      contains(github.event.issue.labels.*.name, 'reQUEST') ||
-      contains(github.event.issue.labels.*.name, 'seQUESTered')
+      github.event.label.name == ':world_map: reQUEST' ||
+      github.event.label.name == ':pushpin: seQUESTered' ||
+      contains(github.event.issue.labels.*.name, ':world_map: reQUEST') ||
+      contains(github.event.issue.labels.*.name, ':pushpin: seQUESTered')
```

The ASP.NET docs repos don't use the emojis in the labels:

```suggestion
      github.event.label.name == 'reQUEST' ||
      github.event.label.name == 'seQUESTered' ||
      contains(github.event.issue.labels.*.name, 'reQUEST') ||
      contains(github.event.issue.labels.*.name, ':pushpin: seQUESTered')
```